examples/Atlas/Atlas.m

```diff
@@ -342,7 +342,7 @@
                                     'drake_instep_shift', 0.0,... % Distance to shift ZMP trajectory inward toward the instep from the center of the foot (m)
                                     'mu', 1.0,... % friction coefficient
                                     'constrain_full_foot_pose', true,... % whether to constrain the swing foot roll and pitch
-                                    'pelvis_height_above_foot_sole', 0.84,... % default pelvis height when walking
+                                    'pelvis_height_above_foot_sole', 0.82,... % default pelvis height when walking
```

Comment:
@kuindersma it turns out that this height does not work well at all for Atlas v3. I've been getting really bad walking performance in the drake unit tests, and when the pelvis height is set to 0.82, a further decrease in height of ~5mm is enough to make the CoM tracking very poor or cause the robot to fall on flat ground. I think we should probably switch this back to 0.84 and also investigate why it's causing a problem.